### ISSUE TITLE
[Doc]: Document runtime compiler requirements for SM70 prebuilt wheels

### ISSUE THREAD

**Issue by yao37750** [1CatAI/1Cat-vLLM-0.0.1#193]
## Documentation gap

  The README recommends prebuilt wheels as the normal installation path and
  describes source-build dependencies separately.

  However, the SM70 FlashQLA path still uses
  `torch.utils.cpp_extension.load()` at runtime:

  https://github.com/1CatAI/1Cat-vLLM/blob/v1.2.2/flash_qla/ops/gated_delta_rule/chunk/sm70/fused_fwd.py

  On a cold Torch extension cache, the first model startup or warmup therefore
  requires a working native CUDA build toolchain even when 1Cat-vLLM was
  installed from a prebuilt wheel.

  In a minimal Ubuntu 24.04 container, the 1Cat-vLLM 1.2.2 wheel installed
  successfully, but the first SM70 extension load could not complete until the
  container also provided:

  - a C/C++ compiler (`c++`, installed through `build-essential`)
  - CUDA Toolkit / `nvcc`
  - CUDA development headers
  - a valid `CUDA_HOME`

  The Python `ninja` package was already installed as a dependency of the
  1Cat-vLLM wheel, so this report is mainly about the system compiler and CUDA
  development toolchain.

  ## Environment

  - Host OS: Ubuntu 20.04
  - Container userspace: Ubuntu 24.04 / glibc 2.39
  - GPU: 2 x Tesla V100-SXM2-32GB (SM70)
  - Tensor parallel size: 2
  - Python: 3.12
  - CUDA Toolkit: 12.8.93
  - PyTorch: 2.10.0+cu128
  - 1Cat-vLLM: 1.2.2
  - TileLang: 0.1.9
  - Model: Qwen3.6-35B-A3B-AWQ

  ## What resolved the environment issue

After configuring NVIDIA's CUDA 12.8 apt repository, the runtime image was extended with:

  ```dockerfile
  RUN apt-get update \
      && DEBIAN_FRONTEND=noninteractive apt-get install --yes \
           build-essential \
           cuda-nvcc-12-8 \
      && rm -rf /var/lib/apt/lists/*

  ENV CUDA_HOME=/usr/local/cuda-12.8
  ENV PATH=/usr/local/cuda-12.8/bin:${PATH}
```

  After the required compiler and CUDA development tools were available, the
  FlashQLA extension could be compiled and loaded from the Torch extensions
  cache.

## Suggested documentation improvement

  Could the prebuilt-wheel Quick Start explicitly explain that some SM70 paths
  perform runtime JIT compilation and therefore still require a native compiler
  and CUDA Toolkit on the first cold-cache startup?

  For example, the environment verification section could include:

  ```bash
  command -v c++
  command -v nvcc
  nvcc --version
  python -c "import ninja; print(ninja.__version__)"
  ```

  It may also be useful to add build-essential to the Ubuntu Quick Start
  installation command.

  ## Optional runtime improvement

  If practical, the SM70 extension loader could check for the compiler and
  nvcc before starting the JIT build and return an actionable error such as:

  ```text
  SM70 FlashQLA cold-cache initialization requires a C++ compiler and nvcc.
  Install build-essential and CUDA Toolkit 12.8, then verify CUDA_HOME.
  ```

  This would make it easier to distinguish a missing build toolchain from a
  kernel, model, or distributed-worker failure.

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.
